backend/app/agents/devrel/github/github_toolkit.py

```
@@ -13,6 +15,16 @@
 from .tools.general_github_help import handle_general_github_help
 logger = logging.getLogger(__name__)
 
+DEFAULT_ORG = os.getenv("GITHUB_ORG")
+
+def normalize_org(org_from_user: str = None) -> str:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)